buildscripts/incremental/setup_conda_environment.cmd

```diff
@@ -21,9 +21,11 @@ call deactivate
 @rem Display root environment (for debugging)
 conda list
 @rem Scipy, CFFI, jinja2 and IPython are optional dependencies, but exercised in the test suite
-conda create -n %CONDA_ENV% -q -y python=%PYTHON% numpy=%NUMPY% cffi pip scipy jinja2 ipython gitpython pyyaml
+conda create -n %CONDA_ENV% -q -y python=%PYTHON% numpy=%NUMPY% cffi pip scipy jinja2 ipython gitpython pyyaml libxml2
 
 call activate %CONDA_ENV%
+@rem Install latest unittest-xml-reporting
+%CONDA_INSTALL% -c conda-forge unittest-xml-reporting
```

Comment:
```suggestion
%CONDA_INSTALL% conda-forge::unittest-xml-reporting
```
so it won't try to change anything to use conda-forge unnecessarily.

Reply:
and that it matches https://github.com/numba/numba/pull/8555/files#diff-319afae18279ecbf9d01cb17a6091e1103ddccadda228ace0a011a5590aa3667R84